### ISSUE TITLE
Remove method not available

### ISSUE THREAD

**Issue by dadebue** [ted-piotrowski/mapbox-gl-shadow-simulator#3]
Hi Ted,

first and foremost: Awesome work, I really love the shade map and I'm thinking about including it in a project!

In the Readme you mention a `remove` method, to remove the shade from the map:
```
// sometime later...
    // ...remove layer
    shadeMap.remove();
```

Unfortunately this method doesn't exist on the class. See screenshot from VSCode:
![Screen 2022-11-30 at 12-45-47](https://user-images.githubusercontent.com/81422785/204788119-ac9cf3aa-8062-49af-8a8c-0911c56ecb47.png)
![Screen 2022-11-30 at 12-46-03](https://user-images.githubusercontent.com/81422785/204788184-2864e5d4-b43f-49b4-82c0-500244585eab.png)

Im importing like this:
`import ShadeMap from 'mapbox-gl-shadow-simulator'`

I'm creating the shade map like this:
```
const shadeMap = new ShadeMap({
  date: new Date(), // display shadows for current date
  color: '#01112f', // shade color
  opacity: 0.7,
   ...
}
```

Am I doing something wrong? Is there another way to remove the shade from the map?

BR
Marco

**Comment by ted-piotrowski** [comment]
Nice catch. I just released version 0.16.0 on NPM, adding the `remove()` method. Let me know if it works for you.

**Comment by dadebue** [comment]
Yep working like a charm. Thanks :)
